projects/core/src/state/utils/processes-loader/processes-loader.reducer.ts

```diff
@@ -40,6 +40,7 @@ export function processesLoaderReducer<T>(
         processesCountDiff &&
         state.processesCount + processesCountDiff < 0
       ) {
+        /* eslint-disable-next-line no-console */
```

Comment:
[MINOR]

It's actually a place in which we'd like to use the LoggerService.

What about adding in a code comment a link to a ticket for this planned change?

Reply:
done in https://github.com/SAP/spartacus/pull/17538